includes/admin/payments/orders.php

```diff
@@ -761,18 +857,26 @@ function edd_order_details_extras( $order = false ) {
 					</div>
 				<?php endif; ?>
 
-				<div class="edd-order-payment-key edd-admin-box-inside">
-					<label for="edd_payment_key" class="label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
-					<input type="text" name="payment_key" id="edd_payment_key" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
+				<div class="edd-admin-box-inside">
+					<div class="edd-form-group">
+						<label for="edd_payment_key" class="edd-form-group__label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
+						<div class="edd-form-group__control">
+							<input type="text" name="payment_key" id="edd_payment_key" class="edd-form-group__input" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
+						</div>
+					</div>
 				</div>
 
 				<div class="edd-order-ip edd-admin-box-inside">
-					<label for="edd_ip" class="label"><?php esc_html_e( 'IP', 'easy-digital-downloads' ); ?></label>
-					<?php if ( edd_is_add_order_page() ) : ?>
-						<input type="text" name="ip" id="edd_ip" value="<?php echo esc_attr( edd_get_ip() ); ?>" />
-					<?php else : ?>
-						<span><?php echo edd_payment_get_ip_address_url( $order->id ); // WPCS: XSS ok. ?></span>
-					<?php endif; ?>
+					<div class="edd-form-group">
+						<label for="edd_ip" class="edd-form-group__label"><?php esc_html_e( 'IP', 'easy-digital-downloads' ); ?></label>
+					<div class="edd-form-group__control">
+						<?php if ( edd_is_add_order_page() ) : ?>
+							<input type="readonly" name="ip" id="edd_ip" class="edd-form-group__input" value="<?php echo esc_attr( edd_get_ip() ); ?>" />
```

Comment:
`type` is still `text`. `readonly` is an attribute in the same way that `type` is, so they exist alongside each other.

https://www.w3schools.com/tags/att_input_readonly.asp

It is a boolean attribute, which means it doesn't need a value (where `text` is the value of the `type` attribte)

Reply:
Sorry for the delay.  I would have sworn that I did this one yesterday.